app/components/apply-page/role-details.tsx

```diff
@@ -0,0 +1,110 @@
+const RoleDetailsSection = () => {
```

Comment:
we should have a role details for software engineer, product designer, and product manager.

So let's take in the what you'll be doing paragraph, the what we're looking for paragraph, and the name of the role + tagline. That way it'll be easy to hook up prismic using this component.